docker/nginx/libs/skynet/account.lua

```diff
@@ -1,13 +1,15 @@
 local _M = {}
 
 -- fallback - remember to keep those updated
-local anon_limits = { ["tierName"] = "anonymous", ["upload"] = 655360, ["download"] = 655360, ["maxUploadSize"] = 1073741824, ["registry"] = 250 }
+local anon_limits = { ["tierID"] = 0, ["tierName"] = "anonymous", ["upload"] = 655360, ["download"] = 655360, ["maxUploadSize"] = 1073741824, ["registry"] = 250 }
 
 -- no limits applied 
-local no_limits = { ["tierName"] = "internal", ["upload"] = 0, ["download"] = 0, ["maxUploadSize"] = 0, ["registry"] = 0 }
+local no_limits = { ["tierID"] = -1, ["tierName"] = "internal", ["upload"] = 0, ["download"] = 0, ["maxUploadSize"] = 0, ["registry"] = 0 }
 
--- free tier name
-local free_tier = "free"
+-- constant tier ids
+local tier_id_internal = -1
```

Comment:
what do we use the `internal` tier for? asking because it might be important to take the internal tier into account in functions like `function _M.has_subscription()`

Reply:
we want to run around the limits for stuff like our website being served from nginx, we dont want to impose any limits on the website

Reply:
removed internal in https://github.com/SkynetLabs/skynet-webportal/pull/1789/commits/4dff19c6c74582b2f4abef195797ea3e61eb695c - we don't run the website off skynet any more and it's redundant now